sequential/seq2pat.py

```diff
@@ -277,11 +277,18 @@ class Seq2Pat:
     sequences : List[list]
         A list of sequences each with a list of events.
         The event values can be all strings or all integers.
+    max_span: Union[int, None]
```

Comment:
Any reason why `Union[int, None]` instead of `Optional[int]`? It accomplishes the same thing either way, but I'm wondering if you had a preference.